pkg/handlers/generic/mutation/imageregistries/credentials/tests/generate_patches.go

```diff
@@ -23,6 +24,11 @@ import (
 
 const (
 	validSecretName = "myregistry-credentials"
+	//nolint:gosec // Does not contain hard coded credentials.
+	cpRegistryCreds = "kubeadmControlPlaneRegistryWithCredentials"
+	//nolint:gosec // Does not contain hard coded credentials.
+	workerRegistryCreds                   = "kubeadmConfigTemplateRegistryWithCreds"
```

Comment:
nit: To be consistent
```suggestion
	cpRegistryCreds = "kubeadmControlPlaneRegistryWithCredentials"
	//nolint:gosec // Does not contain hard coded credentials.
	workerRegistryCreds                   = "kubeadmConfigTemplateRegistryWithCredentials"
```

Reply:
fixed. 9014ba63f9e26c93683d932ce6df6ca77664ca8a